docs/pages/guides/sell-nft.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Selling an NFT'
+---
+
+import { Steps } from 'nextra-theme-docs'
+
+# Selling an NFT
+
+## Introduction
+
+You can create a listing on any NFTs you own directly on the Liteflow dashboard.
+
+## Prerequisites
+
+Before creating an listing, ensure you have:
+
+- Imported or created the collection using the Liteflow dashboard
+- [Set Up the currency you want to use](./setup-currencies).
+- You are owner of the NFT
+
+## Steps
+
+<Steps>
+
+### Access Listings section
+
+Navigate to the [Listings](https://dashboard.liteflow.com/listings) section in your Liteflow dashboard.
+
+### Start creating a listing
+
+Click on the 'Create' button to open the form to create a listing on an NFT.
+
+### Select an NFT
+
+Choose the NFT from the list.
+
+If the NFT is not listed but is listed in the [NFTs](https://dashboard.liteflow.com/nfts) section, it means your wallet is not the owner of this NFT. Make sure your are login with the right wallet.
```

Comment:
Maybe adding this as a warning or info box would be nice

Reply:
added